packages/api/src/controllers/helpers.ts

```diff
@@ -108,6 +108,15 @@ function bytesToHexString(bytes: Uint8Array, separate = false) {
   return result;
 }
 
+export function cleanAdminOnlyFields(
```

Comment:
Was this used anywhere new?

Reply:
Bump! Also think this should be reverted as it's only used in `user` controller. We shouldn't be using this anywhere else. Should use `writeOnly` properties for "private" fields instead.

Reply:
Moved it back to `user.ts`! I moved it while making the simplified response for the `asset` then forgot it there, thanks!